lib/auth/login/widgets/login_form.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+import 'package:form_builder_validators/form_builder_validators.dart';
+import 'package:icons_plus/icons_plus.dart';
+
+class LoginForm extends StatefulWidget {
+  const LoginForm({
+    super.key,
+  });
+
+  @override
+  State<LoginForm> createState() => _LoginFormState();
+}
+
+class _LoginFormState extends State<LoginForm> {
+  final _formKey = GlobalKey<FormBuilderState>(debugLabel: 'login_form');
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<LoginCubit, LoginState>(
+      builder: (context, state) {
+        if (state.email.isNotEmpty) {
+          _formKey.currentState?.fields['email']?.didChange(state.email);
+        }
+
+        return FormBuilder(
+          key: _formKey,
+          child: Padding(
+            padding: const EdgeInsets.symmetric(
+              vertical: FlexSizes.spacerItems,
+            ),
+            child: Column(
+              children: [
+                // Email field
+                FormBuilderTextField(
+                  name: 'email',
+                  decoration: const InputDecoration(
+                    prefixIcon: Icon(LineAwesome.chevron_right_solid),
+                    labelText: 'Email',
+                  ),
+                  initialValue: state.email,
+                  validator: FormBuilderValidators.compose([
+                    FormBuilderValidators.required(),
+                    FormBuilderValidators.email(),
+                  ]),
+                ),
+                const SizedBox(height: FlexSizes.inputFieldSpacing),
+
+                // Password field
+
+                FormBuilderTextField(
+                  name: 'password',
+                  obscureText: state.hidePassword,
+                  decoration: InputDecoration(
+                    prefixIcon: const Icon(LineAwesome.lock_solid),
+                    labelText: 'Password',
+                    suffixIcon: IconButton(
+                      onPressed: () =>
+                          context.read<LoginCubit>().togglePasswordVisibility(),
+                      icon: Icon(
+                        state.hidePassword
+                            ? LineAwesome.eye_slash
+                            : LineAwesome.eye,
+                      ),
+                    ),
+                  ),
+                  validator: FormBuilderValidators.compose([
+                    FormBuilderValidators.required(),
+                  ]),
+                ),
+                const SizedBox(height: FlexSizes.inputFieldSpacing),
+
+                // Remember me and forgot password
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    // Remember me
+                    Row(
+                      children: [
+                        Checkbox(
```

Comment:
How about using `FormBuilderCheckbox` here?

Reply:
A couple of reasons I avoided it:
1. The UI is actually worse (spacing and layout issues)
2. It is tied to the init() state, same as email (which rely on past rememberMe settings)